module/sources/vmware/connection.py

```diff
@@ -1940,7 +1942,7 @@ def add_virtual_machine(self, obj):
 
         if platform is not None:
             vm_data["platform"] = {"name": platform}
-        if annotation is not None:
+        if bool(self.skip_vm_comments) is False and annotation is not None:
```

Comment:
This should stay unchanged. Just add an `annotation = None` above line 1922

Reply:
Like this? 8c2bdf65ad7a82ae797b1e024e4c66cd259eab82